### DISCUSSION TITLE
What is the status of the project?

### DISCUSSION THREAD

**Discussion by nbelyh** [exceljs/exceljs#2194] [Q&A] [answered]
There is not a single PR or commit for the past year (2022) 
What is going on? Where are the maintainers?

**Comment by voliva** [comment]
I'm wondering the same... This library looks more complete than other alternatives (e.g. https://github.com/SheetJS/sheetjs), but the repo status doesn't look healthy at all.

**Reply by nbelyh** [reply]
Yes, this one worked better for me too than sheetjs. But where did the author and maintainers go?
Even if not interested anymore (?), there are numerous contributions and good PRs..

**Reply by zurmokeeper** [reply]
> Yes, this one worked better for me too than sheetjs. But where did the author and maintainers go? Even if not interested anymore (?), there are numerous contributions and good PRs..



> I'm wondering the same... This library looks more complete than other alternatives (e.g. https://github.com/SheetJS/sheetjs), but the repo status doesn't look healthy at all.

Yesterday (23-4-14) I saw that the author has merged some PR, I wonder how far this update can go, will it be a new version?

**Reply by Siemienik** [reply]
Unfortunately, I have no permission to publish npm `exceljs`. Thus I release it as https://www.npmjs.com/package/@siema-team/spreadsheets

As @skypesky mentioned before. It's hard to maintain and develop any OpenSource projects as it's mainly volunteer activity

**Reply by wpyoga** [reply]
Well, this might have been suggested before, but would you (Siemienik) consider appointing someone as a co-maintainer? That way, improvements currently languishing in other forks can be merged into the main project. Anyway, I'm just grateful that you created this project in the first place! :)

**Comment by skypesky** [comment, marked as answer]
I'm glad you raised this question, which is a problem that all open source projects will encounter. The problem is that:

1: The founder of this project created it, and he now pays less attention to this project. We also have a hard time finding him to publish this library to npm, where he rarely appears.
2. In addition, members who have permission to merge pull requests are very busy, and developers never think: who will pay for the time of members of this class library? They have their own work to do and their own hobbies.

**Comment by protobi-pieter** [comment]
https://github.com/protobi/exceljs  (published on npm https://www.npmjs.com/package/@protobi/exceljs )  is a fork that we are maintaining with primary intent to eventually merge back into this master repo https://github.com/exceljs/exceljs

This fork adds support for pivot tables, updates libraries to resolve vulnerability alerts, and includes a number of PRs from this upstream repo with additional test cases.  

Most issues and PRs should be submitted on this master repo https://github.com/exceljs/exceljs which we will occasionally monitor.  Bandwidth for general public support is limited but we can review specific PRs with clean changes and working unit tests, and potentially work on funded requests for specific extensions.
